### PR TITLE
Finally fixing refreshNode

### PR DESCRIPTION
**Pull request by carsten19**
Previous version was buggy (calcName expects other values)

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1283,11 +1283,18 @@ function main() {
...
     });
 
-    zwave.on('value removed', function (nodeID, comClass, instance, index) {
-        adapter.log.debug('value removed: ' + nodeID + ' comClass: ' + JSON.stringify(comClass) + ' instance ' + instance + ' value: '  + JSON.stringify(index));
-
-        const id = calcName(nodeID, comClass, index, instance);
-        delObjects([id]);
+    zwave.on('value removed', function (nodeID, comClass, instance, index) {                
+        adapter.log.debug('value removed: ' + nodeID + ' comClass: ' + JSON.stringify(comClass) + ' instance: ' + instance + ' index: '  + index);
+        var id = calcName(nodeID, comClass);
```
Comment (AlCalzone) [inline main.js]: please use `const`

File: main.js [hunk 2]
```diff
@@ -1283,11 +1283,18 @@ function main() {
...
+        for (var i in objects) {
+            if (!objects.hasOwnProperty(i)) continue;
+            if (i.substring(0, id.length + 1) === id + '.' || i === id) {
```
Comment (AlCalzone) [inline main.js]: `if (i.startsWith(id + '.') || i === id) {`

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Two minor things, looks good otherwise.

**Comment by carsten19** [comment]
done. Please also bump the version, thanks.

**Review by AlCalzone — APPROVED** [review]

**Comment by AlCalzone** [comment]
Released as v1.5.1

**Comment by OleksiyAntonov** [comment]
Do you have plans to update official repo with this version?

**Comment by Apollon77** [comment]
1.5.1 should be in latest in the meantime. Stable is updated after enough users use this version and somit can be considered stable.

**Pull request merged**